sys-libs/minizip-ng/minizip-ng-4.0.5.ebuild

```diff
@@ -52,7 +52,7 @@ multilib_src_configure() {
 		# Compression library options
 		-DMZ_ZLIB=ON
 		-DMZ_BZIP2=ON
-		-DMZ_LZMA=ON
+		-DMZ_LZMA=$(usex lzma)
```

Comment:
Should this really be changing the current behavior of enabling lzma support by default?

Reply:
hm fair question, i was merely following the zstd line when changing it. for me either way is fine. i guess it depends on how its generally handled, useflags mostly disabled to reduce build deps, profiles/users enables the flags when wanted, its my first attempt at contributing to the repo so il leave it up to you guys to decide.

worth knowing tho, libarchive ebuild seems to have the useflag default enabled, meanwhile gdb doesnt. if i adding +lzma to IUSE means its enabled by default? and quite a few others either has it on by default, and not.

Reply:
fwiw USE=lzma is default on all 23.0 profiles now, not that it hurts to make it default in ebuilds if feel it's important

Reply:
if thats the case it doesnt matter much here in my view. so unless someone has any objections il leave it as is. :)